tests/_test_msui/test_mscolab_save_merge_points.py

```diff
@@ -35,11 +35,10 @@
 PORTS = list(range(21000, 21500))
 
 
-# ToDo Understand why this needs to be skipped, it runs when direct called
+@pytest.mark.skip("Uses QTimer, which can break other unrelated tests")
```

Comment:
skipping ok,

why does QTimer or can break other tests, what is the problem with that?

Reply:
The problem is that the QTimer apparently can outlive the test and then run its function when another test already started. This then obviously affects that other test. At least this is what happened in `test_mscolab_merge_waypoints.py` where I was able to get rid of the QTimers (see: https://github.com/Open-MSS/MSS/commit/1e98de5187977fe0e140cf1649776f918b2bf693#diff-5e7ce9c062dc89ba1755f9d8100cb6473831e4575df18b728e922afd641964dc).

Here in `test_mscolab_save_merge_points.py` I was not (yet) able to get rid of the QTimer (and the test was skipped anyway).

Reply:
My thoughts on this: Ideally the test should be independent of other tests when and how it runs. Each test has its own tmp dir and each test has a different server on a different port. They should not actually interfere with each other. We are looking for something that causes a problem, although we make sure that the tests are independent of each other.

Reply:
As I said, my guess is that this is caused by the creation of the QTimer, which then outlives the test in the QApplication and will run at a later point when the Qt event loop is handling events the next time (through processEvents or qWait). At least this is what I saw with the tests in `test_mscolab_merge_waypoints.py`, where the QTimer triggered in the teardown code of a different test.

This test was also skipped for 3 years already, see: https://github.com/Open-MSS/MSS/blame/c1f6e86ce3cd45605825486d1f5a1192d6897503/tests/_test_msui/test_mscolab_save_merge_points.py#L61-L62 (I never saw it not skip based on the if condition there).